packages/text-input/src/TextInput.spec.tsx

```diff
@@ -256,11 +256,18 @@ describe('packages/text-input', () => {
     test('TextInput throws error when neither aria-labelledby or label is supplied', () => {
       // @ts-expect-error
       <TextInput />;
+      // @ts-expect-error
+      <TextInput aria-label="some label" />;
```

Comment:
Not sure about this case @DesignerDave

Reply:
This should be allowed AFAICT

Reply:
👍 Updated to require one of `label`, `aria-label`, or `aria-labelledby` in all cases